### PR TITLE
Use AVA in tests

### PR DESCRIPTION
**Pull request by kevva**
Couldn't stand the old tests :E.

### CODE REVIEW

**Comment by SamVerschueren** [comment]
> Couldn't stand the old tests :E.

Hahaha, know the feeling :p 

The only remark is removing the `should` in every test title. It's a mocha thing...

**Comment by kevva** [comment]
> The only remark is removing the should in every test title. It's a mocha thing...

Yup, I noticed after I looked the third time :P.

File: test.js [hunk 1]
```diff
@@ -1,69 +1,46 @@
...
-	stream.end();
+	t.true(file.contents.length < buf.length);
 });
 
-it('should skip unsupported images', function (cb) {
-	var stream = imagemin();
+test('use custom plugins', async t => {
+	const {stream} = await createFixture({use: [imageminPngquant()]});
+	const compareStream = (await createFixture()).stream;
+	const file = await rfpify(stream.once.bind(stream))('data');
```
Comment (sindresorhus) [inline test.js]: Using `get-stream` might look nicer here? We want all the data from the stream anyways.

Comment (kevva) [inline test.js]: :+1:

Comment (kevva) [inline test.js]: Oh, right, `get-stream` doesn't support objects so this won't work.

Comment (sindresorhus) [inline test.js]: I just realized too. Working on it.

Comment (sindresorhus) [inline test.js]: Done: https://github.com/sindresorhus/get-stream#getstreamarraystream

Comment (kevva) [inline test.js]: Woot. Updated!

File: .travis.yml [hunk 2]
```diff
@@ -4,4 +4,3 @@ node_js:
   - '5'
   - '4'
   - '0.12'
-  - '0.10'
```
Comment (kevva) [inline .travis.yml]: Rather remove this than adding a polyfill. We're going `>=4` soon anyway.

Comment (sindresorhus) [inline .travis.yml]: Yeah. We can just go >=4 when the new imagemin API is used.

**Comment by sindresorhus** [comment]
Awesome! :)

**Pull request merged**